packages/foreman/nodejs-dotenv/nodejs-dotenv.spec

```diff
@@ -1,44 +1,59 @@
 %global npm_name dotenv
-%global enable_tests 1
 
 Name: nodejs-%{npm_name}
 Version: 5.0.1
 Release: 1%{?dist}
-Summary: Loads environment variables from .env for nodejs projects
+Summary: Loads environment variables from
```

Comment:
We really need to improve that summary determination regex. It should consider word bounds rather than just take `.`